ansible/roles/kubernetes-addons/templates/kube-addons.upstart.j2

```diff
@@ -4,7 +4,7 @@ start on started kube-apiserver
 stop on runlevel [!2345]
 
 env TOKEN_DIR={{ kube_token_dir }}
-env KUBECTL_BIN={{ bin_dir }}/kubectl
+env KUBECTL_BIN=/usr/bin/kubectl
```

Comment:
We can't have this hard coded. Please set bin_dir in the associated playbook task if you need to set bin_dir: /usr/bin/  However, I thought /usr/bin/ was already the default for bin_dir.